steps/cadence-genus-synthesis/scripts/read-design.tcl

```diff
@@ -5,5 +5,10 @@
 # Date   : September 28, 2020
 
 read_hdl -sv [lsort [glob -directory inputs -type f *.v *.sv]]
+# Prevent backslashes from being prepended to signal names...
+# this causes SAIF files to be invalid...needs to be before elaboration.
+set_attribute hdl_array_naming_style %s_%d
+set_attribute hdl_bus_wire_naming_style %s_%d
+set_attribute bit_blasted_port_style %s_%d /
 elaborate $design_name
```

Comment:
Super minor, but coding convention wise, can you add newlines before and after the comment block? Also before elaborate since they are semantically in different groups?

Reply:
Done.